src/integer/vec_z/set.rs

```diff
@@ -0,0 +1,108 @@
+// Copyright © 2023 Niklas Siemer
+//
+// This file is part of qFALL-math.
+//
+// qFALL-math is free software: you can redistribute it and/or modify it under
+// the terms of the Mozilla Public License Version 2.0 as published by the
+// Mozilla Foundation. See <https://mozilla.org/en-US/MPL/2.0/>.
+
+//! Implementation to set entries from a [`VecZ`] vector.
+
+use super::VecZ;
+use crate::{error::MathError, integer::Z};
+use std::fmt::Display;
+
+impl VecZ {
+    /// Sets the value of a specific vector entry according to a given `value` of type [`Z`](crate::integer::Z).
+    ///
+    /// Parameters:
+    /// - `entry`: specifies the entry in which the entry is located
+    /// - `value`: specifies the value to which the entry is set
+    ///
+    /// # Example
+    /// ```
+    /// use math::integer::{VecZ, Z};
+    /// use math::utils::VectorDirection;
+    ///
+    /// let mut vector = VecZ::new(5, VectorDirection::ColumnVector).unwrap();
+    /// let value = Z::from_i64(5);
+    /// vector.set_entry(1, value).unwrap();
+    /// ```
+    ///
+    /// # Errors and Failures
+    /// - Returns a [`MathError`] of type [`OutOfBounds`](MathError::OutOfBounds)
+    /// if the given entry outside the vector or negative.
+    pub fn set_entry<S: TryInto<i64> + Display + Copy, T: Into<Z>>(
+        &mut self,
+        entry: S,
+        value: T,
```

Comment:
Use updated version with impl instead of S and T

Reply:
As I'm not able to use the trait and this is the way it is done in MatZ, I do not see what to do differently here. But I also opened an issue for the set_entry and get_entry trait for Vectors.